### ISSUE TITLE
Support to readonly (like disabled)

### ISSUE THREAD

**Issue by rodriformiga** [harvesthq/chosen#2973]
Hello everyone.

I need a support to `readonly `attribute, like `disabled`, but is I set `disabled`, my value is not posted in a form submit.

I found this sample to understand and improve temporary solution.
http://jsfiddle.net/eirc/v2es7L8o/

thanks

**Comment by btr-simbu** [comment]
Yes, the same is needed for me too. I'm using Chosen for nearly around 3 years but I can't find any solution to use Readonly attribute. It'd be great if an option to make the field readonly instead of disabling it.

**Comment by Cristianpl4y** [comment]
Até hoje nada ??
